packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2179,6 +2195,22 @@ class SemanticsProperties extends DiagnosticableTree {
   /// {@endtemplate}
   final SemanticsInputType? inputType;
 
+  /// {@template flutter.semantics.SemanticsProperties.maxValue}
+  /// The maximum value of the node.
+  ///
+  /// {@endtemplate}
+  ///
+  /// {@macro flutter.semantics.SemanticsProperties.maxValue}
```

Comment:
you don't need to re macro here, the doc in `@template` in this section is still visible when compiling the doc. using macro here will cause it to duplicate the same doc.

Reply:
The doc should also mention this is used in conjunction with `[value]`. and a typical usage is a progress indicator